src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public abstract class Machine {
+    public abstract void doWork();
+
+    public abstract void stopWork();
+
```

Comment:
redundant blank line
```suggestion
```

Reply:
it's for checkstyle, otherwise project won't build

Reply:
or will build with errors